src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -127,7 +128,8 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
 
         copyCommand += " \"%s\" " % sourcePFN
         copyCommand += " \"%s\" \n" % targetPFN
-
+        copyCommand += "RC=$? \n"
+        copyCommand += "echo \"xrdcp exit code: $RC\" \n"
```

Comment:
It makes no difference, but you would not have to escape it if written as (single quotes outside):
```
        copyCommand += 'echo "xrdcp exit code: $RC" \n'
```

Reply:
That's true, but I wanted to keep it consistent with the other lines. If we pursue this approach, we should apply it to every line in the script. What do you think @amaltaro?

Reply:
Nahh, it is not worth it. But as we look into these stage out implementations in Stephan's ticket, we might consider that.